JsonFormReadme.md

```diff
@@ -0,0 +1,61 @@
+
+-----------------------------------------------------------------------------------------------
+# documentation   
+
+## required attributes
+
+1) key - a key is always string and enclosed in quotation mark
+
+2) name - specify the name of a feild
+
+3) label - used to create caption for a form control
+
+4) id - specify the unique id for an element
+
+5) type - defines the feild
+
+6) placeholder - shows information belongs in a feild
+
+7) validation - this value is a function that return a string errort or null if the feild is valid
+
+8) validation_server - it needs to validate on the server in addtion to on the client
+
+9) class - 
+
+10) wrapper class - Its provide all the of the componnets attribute and properties as props on
+
+the react component
+ 
+
+## validation
+
+1) required - this validates that the field has been provided and cannot be empty.
+
+2) min - this is the constraint to be put on the value and represents minimum acceptable value. 
+
+3) max - this is the constraint to be put on the value and represents maximum acceptable value. 
+
+4) Item - defines what appears in the array 
```

Comment:
@sajana1829 What appears in what array? Please put a description that makes sense to someone who is new to the project.
`Item - defines the options available in a radio button.` or something similar

Reply:
@sajana1829 Also is `Item` the correct name for the validation?

Reply:
item is a validation keyword, we can use like minItems, uniqueItems.

Reply:
@sajana1829 We didn't implement these in our code???